tests/core/vm/test_vm_state.py

```diff
@@ -74,3 +74,31 @@ def test_set_storage_input_validation(state, address, slot, new_value):
 def test_delete_storage_input_validation(state):
     with pytest.raises(ValidationError):
         state.delete_storage(INVALID_ADDRESS)
+
+
+# TODO candidate to build a core ethereum test (surprising that this case wasn't covered, already!)
```

Comment:
Should we turn this into an issue at ethereum/tests and link to this PR?

Reply:
cc @holiman

Reply:
I just wrote it up in #1867